crates/services/txpool/src/service.rs

```diff
@@ -254,6 +250,7 @@ where
 
                     let acceptance = match checked_tx {
                         Ok(tx) => {
+                            let id = tx.transaction().cached_id().expect("`Checked` tx should have cached id");
```

Comment:
It is the `expect` because this kind of information should be available now. But because we forgot to expose it, we don't have access right now. I don't want to use `?` because it will hide the problem if, at some point, this information is not available.